client/landing/jetpack-cloud/components/activity-card/index.jsx

```diff
@@ -171,11 +179,27 @@ class ActivityCard extends Component {
 					className="activity-card__popover"
 				>
 					<Button
-						href={ backupRestorePath( siteSlug, activity.rewindId ) }
+						href={ ! doesRewindNeedCredentials && backupRestorePath( siteSlug, activity.rewindId ) }
 						className="activity-card__restore-button"
+						disabled={ doesRewindNeedCredentials }
 					>
 						{ translate( 'Restore to this point' ) }
 					</Button>
+					{ doesRewindNeedCredentials && (
+						<div className="activity-card__credentials-warning">
+							<img src={ missingCredentialsIcon } alt="" role="presentation" />
+							<div className="activity-card__credentials-warning-text">
+								{ translate(
+									'{{a}}Enter your server credentials{{/a}} to enable one-click restores from your backups.',
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **17** times:
`translate( 'Enter your server credentials to enable one-click restores for Backups. {{a}}Find your server credentials{{/a}}' )` *ES Score: 6*